shell/probe/nocturnal.sh

```diff
@@ -7,7 +7,7 @@ dos=$(echo $sys | tr '[:upper:]' '[:lower:]')
 
 while :
 do
-    temp=$(mktemp)
+    if [ -z "$PRELUDE_DIR" ];then temp=$(mktemp); else temp="${PRELUDE_DIR}/$(uuidgen)"; fi
```

Comment:
Logic is too complex. I'm also not so sure how uuidgen will hold up on all distos.

Reply:
what about `$RANDOM`  as the test name ? I'm under the assumption that it's a bad idea to choose a static filename (or filename prefix) like "prelude-vst" b/c that tips off EDRs - is that right?

Reply:
I'm not worried about the name per say, as if they squash our file and it's not malicious, we'll report as such. 

That said, if the probe was written to a writable directory via the installer, does the probe need to know about the location or can it just write tests to its cwd?

Reply:
I assume in most cases, users won't be setting PRELUDE_DIR, and for those, it'd be nice to keep writing to /tmp to clean up left over VSTs (from error exits)

If that's not the case/ we don't care about that, I can remove the if/else here and just have the VSTs always writing to cwd (and PRELUDE_DIR is removed as an env variable). It's up to the user to either run our installer with PRELUDE_DIR or download the probe themselves to a writable/executable location

Reply:
This would make sense to me. Pack the intelligence of where to put the probe in the installer and if someone skips using it, they'd need to assume that knowledge.

Reply:
okee. updated so that
- install scripts have PRELUDE_DIR env var option to override the default probe install directory we choose (`/opt` , or `~` generally)
- probes all download VSTs to cwd

tested:
- hades on darwin m1
- install.sh & nocturnal on darwin m1 and ubuntu
- install.ps1 and raindrop on windows 2019 (i think)